assets/css/sensei-course-theme.scss

```diff
@@ -0,0 +1,12 @@
+@import 'inter-font';
```

Comment:
Had to move the main `sensei-course-theme.scss` CSS file out to the `css/` directory, because I think we haven't figured out how to make relative font URLs work otherwise.